### ISSUE TITLE
abuseipdb➡minimum_confidence_score: (1 to 100) 0?

### ISSUE THREAD

**Issue by yvrdarb** [CIDRAM/CIDRAM#224]
For the abuse db confidence score, a value between 1 and 100 is specified; 0 is a valid score on abuse db which indicates a once active ip address that has been de-ranked over time.

Without testing at 0, any foreseeable issues?

**Comment by Maikuolan** [comment]
Really..? I'd thought that the API would always be returning a confidence score, no matter whether an IP was ever listed or not. I'll test this out and reply back a little later.

**Comment by yvrdarb** [comment]
![Capture](https://user-images.githubusercontent.com/55927900/118014764-65905480-b308-11eb-83e0-3b7224f06c39.PNG)

**Comment by Maikuolan** [comment]
I just checked the raw results from the API from checking against my own IP address (which has zero reports against it at AbuseIPDB), and the confidence score still appears (0, of course, since no reports), so I guess, the confidence score is standard, and will always appear.

But, I also tested some IPs which are whitelisted at AbuseIPDB, but which have some reports against them, to confirm the effects there, and they all also show as having a 0 confidence score, but with recent, active reports against them.

I think, if users specify 0 as the minimum confidence score required for the AbuseIPDB module, it'll cause *all* requests to be indiscriminately blocked by their installation, regardless of whether they are, or have been listed before, so probably not really a good idea, nor what we really want here (though I would also note, that the so called required range actually isn't being *enforced* by CIDRAM at the moment, so users *could* still set 0 right now, if they really wanted to, though I still wouldn't recommend it).

Maybe, instead, we just need a new configuration option added, to enable directly checking the actual number of reports, in addition to the existing checks for the confidence score?
